components/ide/jetbrains/gateway-plugin/gradle-latest.properties

```diff
@@ -1,10 +1,10 @@
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild=241.17011
-pluginUntilBuild=241.*
+pluginSinceBuild=242.20224
+pluginUntilBuild=242.*
 # Plugin Verifier integration -> https://github.com/JetBrains/gradle-intellij-plugin#plugin-verifier-dsl
 # See https://jb.gg/intellij-platform-builds-list for available build versions.
-pluginVerifierIdeVersions=2024.1
-# Version from "com.jetbrains.gateway" which can be found at https://www.jetbrains.com/intellij-repository/snapshots
-platformVersion=241.17011-EAP-CANDIDATE-SNAPSHOT
+pluginVerifierIdeVersions=2024.2
+# Version from "com.jetbrains.gateway" which can be found at https://www.jetbrains.com/updates/updates.xml
+platformVersion=242.20224.92
```

Comment:
won't manual changes break auto update process?

Reply:
ok, we will need to update our automation for latest then?

Reply:
> ok, we will need to update our automation for latest then?

Yes